src/app/project/ontology/ontology.component.ts

```diff
@@ -169,61 +169,69 @@ export class OntologyComponent implements OnInit {
         if (this.beta) {
             const uuid = this._route.parent.snapshot.params.uuid;
             this._route.params.subscribe(params => {
+                this.loading = true;
                 this._dspApiConnection.admin.projectsEndpoint.getProjectByIri(this._projectService.uuidToIri(uuid)).subscribe(
                     (res: ApiResponseData<ProjectResponse>) => {
                         const shortcode = res.body.project.shortcode;
                         const iriBase = this._ontologyService.getIriBaseUrl();
                         const ontologyName = params['onto'];
                         this.ontologyIri = `${iriBase}/ontology/${shortcode}/${ontologyName}/v2`;
 
-                        this.disableContent = (window.innerWidth <= 768);
+                        this.initView();
+                    }
+                );
+            });
+        } else {
+            // to be removed once the beta view is implemented
+            this.initView();
+        }
+    }
 
-                        // get information about the logged-in user
-                        this.session = this._session.getSession();
+    initView(): void {
+        this.disableContent = (window.innerWidth <= 768);
 
-                        // is the logged-in user system admin?
-                        this.sysAdmin = this.session.user.sysAdmin;
+        // get information about the logged-in user
+        this.session = this._session.getSession();
```

Comment:
It is confusing to have two parameters "session" and "_session". Maybe renaming the second one to "_sessionService" makes it more clear

Reply:
fair point, done in [d3621ad](https://github.com/dasch-swiss/dsp-app/pull/891/commits/d3621ad9e5f5316540202c6720e60d42fd7fd382)